small-talk/pages/api/auth2.js

```diff
@@ -0,0 +1,30 @@
+import Mongoboi from "@/db/mongo"
+import { headers } from "next/headers"
+import { NextResponse } from "next/server";
+
+export default async function POST(req,res) 
+{
+    let session_id;
+    console.log(req.headers)
+    if (req.headers.authorization) 
+    {
+        session_id = req.headers.authorization
+        console.log("AUTH2: "+ session_id)
+    }
+    else
+    {
+        res.status(200).json({ found: false }); return;
+    }
+
+    const uri = "mongodb+srv://" + process.env.DB_USER + ":" + process.env.DB_PASS + "@smalltalkcluster0.jo4jne6.mongodb.net/?retryWrites=true&w=majority"
```

Comment:
This makes me realize we currently have the user and pass hard-coded into a const right now. Should definitely change that 😅

Reply:
Created a new issue for this here: https://github.com/UNLV-CS472-672/2024-S-GROUP4-SMTK/issues/90

Reply:
We should have the entire uri split into components that can be used to build a complete string. I think it's kinda weird to have the uri in plaintext, but it may not even be a big deal in the end.